src/setupTests.js

```diff
@@ -1,6 +1,6 @@
 import 'jest-styled-components';
 import 'jest-dom/extend-expect';
```

Comment:
I believe jest-dom was also moved if I'm not mistaken

Reply:
It seems we still need that otherwise some extensions of `expect` we're using won't be there:
```
TypeError: expect(...).toHaveFocus is not a function
```

Reply:
that's weird cause it looks like it's there https://github.com/testing-library/jest-dom